src/proximity/Proximity.style.tsx

```diff
@@ -1,16 +1,20 @@
 import styled from 'styled-components'
 
 import { space } from '../_utils/branding'
+import { TextCaption } from '../typography/caption'
 
 export const StyledProximity = styled.div`
-  display: inline-block;
+  display: flex;
   line-height: 0;
-
-  svg {
-    margin-top: ${space.s};
-  }
+  align-items: center;
+  padding-top: ${space.s};
```

Comment:
Have you checked if theses changes are  breaking the current `Itinerary`?

Reply:
Yes that's good